packages/runtime/runtime-definitions/src/protocol.ts

```diff
@@ -27,6 +27,11 @@ export interface ISignalEnvelope {
     address?: string;
 
     /**
+     * Identifier for the signal being submitted.
+     */
+    clientSignalSequenceNumber: number;
```

Comment:
i don't believe this breaking change is safe, as there can be existing clients sending singals without this set. i believe at a minimum is should be optional. secondarily, in protocol defs signals already have an optional reference sequence number. this seem redundant

Reply:
https://github.com/microsoft/FluidFramework/blob/a1585eed1dd54f1d55e9c7be5a93167ee28609de/common/lib/protocol-definitions/src/protocol.ts#L193

Reply:
yes, I'm not seeing, in my test cases, the referenceSequenceNumber being used and I believe we want to control the lifetime of the sequence locally.    As for the optional, I had the same assumption, but Vlad had a good point as we guarantee the message was sent by the same client ( message.clientId === this.clientId )
"I do not think envelope.clientSignalSequenceNumber !== undefined check is required - if it was sent by this client, then it has to have a sequence number. "